app/src/api/types.ts

```diff
@@ -156,6 +156,11 @@ export enum DeployStatus {
   Failed = 'Failed',
 }
 
+export enum TableOrder {
```

Comment:
I think this is probably overkill. We could accomplish the same result (strict typing to only two strings) but doing `type TableOrder = 'asc' | 'desc'`

Reply:
And also I believe there is a type for this somewhere in our codebase already.